rosco-manifests/src/main/java/com/netflix/spinnaker/rosco/manifests/TemplateUtils.java

```diff
@@ -30,6 +28,8 @@
   @Autowired
   ClouddriverService clouddriverService;
 
+  private static final String PATH_SEPARATOR = "contents/";
```

Comment:
This may not be true for artifacts stored in Bitbucket or other supported SCMs. Let's see if we can come up with a way to do this without depending on Github's API URL format.

Reply:
removed, now the logic is based on the full path instead of `/contents`